packages/web/package.json

```diff
@@ -268,10 +268,9 @@
     "typescript": "5.0.4",
     "typescript-plugin-css-modules": "^3.4.0",
     "url": "^0.11.0",
-    "vite": "4.5.0",
+    "vite": "^4.2.2",
```

Comment:
should we pin this? why the move down?

Reply:
This was to use tsconfig-paths, but we can't use that with vike anyway so I'll revert this change